CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # BigFlow changelog
 
+## Version 1.7.0
+
+### Changed
+
+* Bumped dependencies of main libraries (eg. Apache Beam to version 2.45 or BigQuery to version 3.6.0). It enabled compatibility with MacBooks with M1 processor. 
```

Comment:
I don't see these changes in `requirements.in` file. This is suspicious because if there were no changes to requirements.in then why requirements.txt has been changed?